### PR TITLE
Support updating lockfiles for pnpm and yarn

### PR DESCRIPTION
**Pull request by frantic1048**
This PR fixes issue #769 and also adds support for pnpm lockfile updates.

### CODE REVIEW

**Comment by frantic1048** [comment]
@microsoft-github-policy-service agree company="RightCapital"

File: src/bump/performBump.ts [hunk 1]
```diff
@@ -52,6 +54,33 @@ export async function updatePackageLock(cwd: string): Promise<void> {
...
       console.warn('Updating package-lock.json failed. Continuing...');
     }
+  } else if (root && fs.existsSync(path.join(root, 'pnpm-lock.yaml'))) {
+    console.log('Updating pnpm-lock.yaml after bumping packages');
+    const res = await pnpm(['install', '--lockfile-only', '--ignore-scripts'], { stdio: 'inherit' });
+    if (!res.success) {
+      console.warn('Updating pnpm-lock.yaml failed. Continuing...');
+    }
+  } else if (root && fs.existsSync(path.join(root, 'yarn.lock'))) {
+    console.log('Updating yarn.lock after bumping packages');
```
Comment (ecraig12345) [inline src/bump/performBump.ts:64]: The lock file for yarn v1 doesn't reflect local packages, so this step should be skipped for yarn v1.

**Review by ecraig12345 — COMMENTED** [review]
Thanks for the contribution! This makes sense, just needs a change for yarn 1.

**Review by ecraig12345 — APPROVED** [review]
Thanks!

**Pull request merged**